src/web-api.js

```diff
@@ -151,38 +152,32 @@ class WebApi {
 
         _private.data[key] = bool;
       },
-      determineSecKey: function(timestamp) {
-        // calculate an outgoing signature
-        return new Signature(_private.data.partner_id, _private.data.api_key).generate_sec_key(timestamp || _private.data.timestamp);
-      },
       determineSignature: function(timestamp) {
```

Comment:
again function serves no purpose anymore, so let's get rid of it and inline the signature creation on ;ine 168